composer.json

```diff
@@ -35,14 +32,14 @@
 			"composer dump-autoload"
 		],
 		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l",
-		"cs:check": "php-cs-fixer fix --dry-run --diff",
-		"cs:fix": "php-cs-fixer fix",
-		"openapi": "generate-spec",
-		"psalm": "psalm --threads=1",
-		"psalm:dev": "psalm --no-cache --threads=$(nproc)",
-		"psalm:update-baseline": "psalm --threads=1 --update-baseline",
-		"psalm:clear": "psalm --clear-cache && psalm --clear-global-cache",
-		"psalm:fix": "psalm --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
+		"cs:check": "vendor/bin/php-cs-fixer fix --dry-run --diff",
+		"cs:fix": "vendor/bin/php-cs-fixer fix",
+		"openapi": "vendor/bin/generate-spec",
+		"psalm": "vendor/bin/psalm --threads=1",
+		"psalm:dev": "vendor/bin/psalm --no-cache --threads=$(nproc)",
+		"psalm:update-baseline": "vendor/bin/psalm --threads=1 --update-baseline",
+		"psalm:clear": "vendor/bin/psalm --clear-cache && psalm --clear-global-cache",
+		"psalm:fix": "vendor/bin/psalm --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
```

Comment:
This is not necessary, the binaries are still resolved correctly

Reply:
Ah, origined from a problem in psalm.xml, this was my first change to try to fix it. But right